src/amo/components/UserProfileEdit/styles.scss

```diff
@@ -86,6 +86,13 @@ $font-size-aside: 10px;
   }
 }
 
+.UserProfileEdit textarea {
```

Comment:
Could you use `.UserProfileEdit-biography` instead?

Reply:
Updated!